mev-rs/src/blinded_block_relayer/api/client.rs

```diff
@@ -24,8 +24,12 @@ impl BlindedBlockRelayer for Client {
     }
 
     // TODO support content types
-    async fn submit_bid(&self, signed_submission: &SignedBidSubmission) -> Result<(), Error> {
-        let response = self.api.http_post("/relay/v1/builder/blocks", signed_submission).await?;
+    async fn submit_bid(&self, signed_submission: &SignedBidSubmission, with_cancellations: bool) -> Result<(), Error> {
+        let path = format!("/relay/v1/builder/blocks");
+        let target = self.api.endpoint.join(&path).map_err(ApiError::from)?;
+        let mut request = self.api.http.post(target);
+        if with_cancellations { request = request.query(&[("cancellations", with_cancellations)])};
```

Comment:
it looks like the api takes the value `1` instead of the boolean `true`

ref:
https://flashbots.github.io/relay-specs/#/Builder/submitBlock